resources/js/Pages/Praktikan.vue

```diff
@@ -1449,6 +1448,12 @@ export default {
         return globe.badScoreText[Math.floor(Math.random() * globe.badScoreText.length)];
     },
 
+    tpPickRandomProgram: function()
+    {
+        const globe = this;
+        return globe.randomNumber = Math.floor(Math.random()*3)+5;
```

Comment:
Change this function like this for efficiency

```suggestion
        return Math.floor(Math.random()*3)+5;
```